nilearn/decomposition/canica.py

```diff
@@ -178,6 +217,8 @@ def _unmix_components(self):
         for component in self.components_:
             if component.max() < -component.min():
                 component *= -1
+        if hasattr(self, "masker_"):
+            self.components_img_ = self.masker_.inverse_transform(self.components_)
```

Comment:
This is great! Can you also propagate the change to the examples (and add a comment above the use of components_img_ exists only in 0.4.1 and after, and that users should unmask 'components_' for older versions).